cmd/snap-bootstrap/initramfs_systemd_mount.go

```diff
@@ -75,6 +75,27 @@ type systemdMountOptions struct {
 	Private bool
 	// Umount the mountpoint
 	Umount bool
+	// Overlayfs indicates an overlay filesystem.
+	Overlayfs bool
+	// Directories to be used as lower layers of an overlay mount.
+	// It does not need to be on a writable filesystem.
+	LowerDirs []string
+	// A directory to be used as the upper layer of an overlay mount.
+	// This is normally on a writable filesystem.
+	UpperDir string
+	// A directory to be used as the workdir of an overlay mount.
+	// This needs to be an empty directory on the same filesystem as upperdir.
+	WorkDir string
+}
+
+func pathContainsAny(i string, chars string) bool {
```

Comment:
what's the difference of this with strings.ContainsAny?

Reply:
None. Thanks for pointing it out, I removed it in favor of `strings.ContainsAny` and introduced 2 separate const character sets. That should address the other comments as well.

So I had initially written `pathContainsAny` to contain the characters that we forbid but I forgot to change it back when I had to make it more generic, as I needed to call it with a different character set in order to relax the check for overlayfs's lowerdir option (which could contain `:`)